CONTRIBUTING-development.md

```diff
@@ -0,0 +1,150 @@
+# Coding style
+
+* We almost always use **CamelCase**, when naming functions, variables etc., **not snake_case**.
```

Comment:
I think this should contain a section at the start about citus_indent, there's currently a good one on this in CONTRIBUTING.md. Let's move that to the top of STYLEGUIDE.md instead.